install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -742,7 +742,8 @@ label.
 This means two things for unprivileged containers:
 
 - The volume will be given a `type` which is accessible by unprivileged
-containers. This `type` is usually *svirt_sandbox_file_t*.
+containers. Previously, the label specified was `svirt_sandbox_file_t`. This label
```

Comment:
@pmorie I assume you mean we are using the container_file-t label here? So, something like:
The volume will be given a type which is accessible by unprivileged containers. **This type is usually container_file_t, which treats volume as container content.** Previously, the label specified wassvirt_sandbox_file_t`. This label
is xref:../../release_notes/ocp_3_5_release_notes.adoc#ocp-35-kubelet-directory-should-not-share-selinux-labels-with-its-containers[no longer used due to security concerns].